### ISSUE TITLE
Test vs Validation sets

### ISSUE THREAD

**Issue by Emirismail** [Forest-art/LSV-LP#4]
Hello,

I am writing in reference to your paper, specifically the section titled "Results and Analysis on the Validation Set." Given that the LSV-LP dataset has three subsets (train, validation, and test), the term "validation" as used in your paper is a bit ambiguous. Could you please clarify whether the results presented in Table 5 are based on the test set or the validation set of the LSV-LP dataset?

Many thanks for your time.
